Gemfile

```diff
@@ -13,11 +13,16 @@ end
 gem 'rake', :group => [:development, :test]
 gem 'jira-ruby', :group => :development
 
+beaker_version = ENV['BEAKER_LOCATION'] || ENV['BEAKER_VERSION']
```

Comment:
We are standardizing on BEAKER_VERSION and BEAKER_RSPEC_VERSION - it's already integrated into a number of other projects.